dao-factory/src/main/java/com/iluwatar/daofactory/PostgresDAOFactory.java

```diff
@@ -0,0 +1,45 @@
+package com.iluwatar.daofactory;
+
+import java.sql.*;
+
+/**
+ * A DAO implementation for the Postgres Factory
+ */
+public class PostgresDAOFactory extends DAOFactory {
+    // Connection variable information to Postgres database
+    private static final String conn = "jdbc:postgresql://localhost:5432/dao_factory";
+    private static final String user = "postgres";
+    private static final String pass = "123";
```

Comment:
If I understood correctly, running the example requires that a local PostgreSQL database server is running? Usually what we do is use an in-memory database (like H2) by default and then it's possible to configure the code to use an external database, but not required.